lib/fathead/sass/parse_functions.py

```diff
@@ -295,9 +297,9 @@ def create_file(self):
                     list_of_data = [
                         name,                       # unique name
                         'A',                        # type is article
-                        name + ' function',          # redirect data
+                        name + ' function',         # redirect data
```

Comment:
This still needs to be removed, you can't provide redirect data for an article (i.e. `'A'`)